Tests/Block/PageListBlockServiceTest.php

```diff
@@ -29,7 +29,7 @@ protected function setUp()
     {
         parent::setUp();
 
-        $this->pageManager = $this->getMock('Sonata\PageBundle\Model\PageManagerInterface');
+        $this->pageManager = $this->createMock('Sonata\PageBundle\Model\PageManagerInterface');
```

Comment:
It doesn't on old versions indeed, let me find out what to use as a new requirement